src/main/features/claim/views/check-and-send.njk

```diff
@@ -74,21 +76,20 @@
 
     {{ tableStart('claimDetails', 'Claim details') }}
     {{ row('Why you’re owed the money:', draftClaim.reason.reason, paths.reasonPage.uri ) }}
-
-    {{ row('Timeline of what happened:', '', paths.timelinePage.uri, false ) }}
-    {% set allTimelineRows = draftClaim.timeline.getPopulatedRowsOnly() %}
-    {% set lastItemIndex = allTimelineRows.length - 1 %}
-    {% for timelineRow in allTimelineRows %}
-      {{ row(timelineRow.date, timelineRow.description, '', loop.index0 === lastItemIndex) }}
-    {% endfor %}
-
-    {{ row('Your evidence (optional):', '', paths.evidencePage.uri, false ) }}
-    {% set allEvidenceRows = draftClaim.evidence.getPopulatedRowsOnly() %}
-    {% set lastItemIndex = allEvidenceRows.length - 1 %}
-    {% for evidenceRow in allEvidenceRows %}
-      {{ row(evidenceRow.type.displayValue, evidenceRow.description | default(''), '', loop.index0 === lastItemIndex) }}
-    {% endfor %}
-
+    {{
+      timelineCheckAndSend(
+        header = 'Timeline of what happened',
```

Comment:
No colon at the end? There are still some places with colon so I wonder what is the approach. The same in other refactored places.

Reply:
No, all colons should be removed from check and send pages according to latest pattern